climakitae/core/data_interface.py

```diff
@@ -946,122 +1037,148 @@ def _update_unit_options(self):
+            pass
 
     @param.depends(
+        "approach",
         "scenario_ssp",
         "scenario_historical",
         "downscaling_method",
         "time_slice",
         watch=True,
     )
     def _update_data_warning(self):
-        """Update warning raised to user based on their data selections."""
+        """Update warning raised to user based on their data selections.
+        No warming shown if approach is set to warming level.
+        """
         data_warning = ""
         bad_time_slice_warning = """You've selected a time slice that is outside the temporal range 
         of the selected data."""
 
-        # Set time range of historical data
-        if self.downscaling_method == "Statistical":
-            historical_climate_range = self.historical_climate_range_loca
-        elif self.downscaling_method == "Dynamical+Statistical":
-            historical_climate_range = self.historical_climate_range_wrf_and_loca
-        else:
-            historical_climate_range = self.historical_climate_range_wrf
-
-        # Warning based on data scenario selections
-        if (  # Warn user that they cannot have SSP data and ERA5-WRF data
-            True in ["SSP" in one for one in self.scenario_ssp]
-        ) and ("Historical Reconstruction" in self.scenario_historical):
-            data_warning = """Historical Reconstruction data is not available with SSP data.
-            Try using the Historical Climate data instead."""
+        if self.approach == "Warming Level":
+            data_warning = ""
 
-        elif (  # Warn user if no data is selected
-            not True in ["SSP" in one for one in self.scenario_ssp]
-        ) and (not True in ["Historical" in one for one in self.scenario_historical]):
-            data_warning = "Please select as least one dataset."
+        else:
+            # Set time range of historical data
+            if self.downscaling_method == "Statistical":
+                historical_climate_range = self.historical_climate_range_loca
+            elif self.downscaling_method == "Dynamical+Statistical":
+                historical_climate_range = self.historical_climate_range_wrf_and_loca
+            else:
+                historical_climate_range = self.historical_climate_range_wrf
+
+            # Warning based on data scenario selections
+            if (  # Warn user that they cannot have SSP data and ERA5-WRF data
+                True in ["SSP" in one for one in self.scenario_ssp]
+            ) and ("Historical Reconstruction" in self.scenario_historical):
+                data_warning = """Historical Reconstruction data is not available with SSP data.
+                Try using the Historical Climate data instead."""
+
+            elif (
+                (  # Warn user if no data is selected
+                    all("SSP" not in one for one in self.scenario_ssp)
+                )
+                and (
+                    not True
```

Comment:
sorry, i missed this the first time. this could also use all, to something like `all("Historical" not in one for one in self.scenario_historical)`

Reply:
leaving as-is for now but we can change in a future PR